### DISCUSSION TITLE
How do I keep <menu> from being cut off?

### DISCUSSION THREAD

**Discussion by lucassilvas1** [beercss/beercss#311] [Q&A] [answered]
Hi,

How should you go about keeping `<menu>` elements from being cut off by containers? I have a container with a fixed width, and it cuts off a part of the menus. I've tried adding `overflow-x: visible` to every parent, but it didn't help.

![image](https://github.com/beercss/beercss/assets/52336985/a0a1595b-3645-4018-8e13-b44b3edd5c9a)

**Comment by leonardorafael** [comment, marked as answer]
Hi @lucassilvas1 

This is a browser behavior. When you have a container with `overflow` (any property or value defined), all child elements will be rendered inside it (including `position: absolute` elements, same as the `<menu>`).

This codepen shows the behavior https://codepen.io/leo-bnu/pen/VwOKKjQ.

**Reply by lucassilvas1** [reply]
Thanks, it was cutting the menu off even though I hadn't set `overflow-x` to anything, setting it to `visible` or `auto` didn't work either. I had to remove `overflow-y` for it to work.
